app/controllers/interviews_controller.rb

```diff
@@ -0,0 +1,74 @@
+class InterviewsController < ApplicationController
+	def index
+	  @interviews = Interview.order('created_at DESC')
+	end
+
+	def new
+	  @interview= Interview.new
+	  @users=User.all
+      @selected=Array.new
```

Comment:
Inconsistent spacing
```suggestion
	  @interview = Interview.new
	  @users =User.all
         @selected=Array.new
```